projects/cdk/pipes/repeat-times/index.ts

```diff
@@ -0,0 +1,14 @@
+import {Pipe, type PipeTransform} from '@angular/core';
+import type {TuiIterationRange} from '@taiga-ui/cdk/types';
+
+type Range<From extends number, To extends number> = TuiIterationRange<From, To, 1>;
+
+@Pipe({
+    standalone: true,
+    name: 'tuiRepeatTimes',
+})
+export class TuiRepeatTimesPipe implements PipeTransform {
+    public transform<N extends number>(length: N): Range<1, N> {
```

Comment:
Should `0` be a valid option as minimum value of range ?
```suggestion
    public transform<N extends number>(length: N): Range<0, N> {
```

It can useful for cases with dynamic show/hide logic

Reply:
I can't use `Range<0, N - 1>`

Reply:
fixed